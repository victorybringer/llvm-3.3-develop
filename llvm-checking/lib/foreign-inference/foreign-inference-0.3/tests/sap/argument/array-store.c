struct S {
  int *p1[100];
  int *p2[5];
};

void f(struct S *s, int *x, int ix) {
  s->p2[ix] = x;
}
