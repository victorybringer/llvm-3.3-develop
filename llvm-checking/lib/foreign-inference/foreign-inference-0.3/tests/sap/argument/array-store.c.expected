fromList [ (("f", "x"), fromList [(0, "%struct.S*", [AccessDeref, AccessField 1,AccessArray])])
         ]